src/main/resources/application.conf

```diff
@@ -254,6 +271,35 @@ oauth2 {
     clientSecretFile = ${?MICROSOFT_CLIENT_SECRET_FILE}
     scopes = "user.read"
   }
+
+  oauth2Api {
```

Comment:
i think you should separate config for each provider (but the jwt and escher can be one common)